cds_ils/circulation/indexer.py

```diff
@@ -18,9 +19,14 @@ def index_extra_fields_for_loan(loan_dict):
     needed when fetching the loan details.
     """
     document_class = current_app_ils.document_record_cls
-    document_record = document_class.get_record_by_pid(
-        loan_dict.get("document_pid")
-    )
+    try:
+        document_record = document_class.get_record_by_pid(
```

Comment:
@Pineirin I didn't double-check, but from what I recall, I think that we saw that it might happen just after the `get_loans`, when checking for referenced relations?